test/webRDFS/generate_successful_delete.py

```diff
@@ -0,0 +1 @@
+print("{\"boolean\":true}")
```

Comment:
It's probably better to put the expected output in a plain text file as a test fixture, instead of invoking Python just to do a print. See `echo -e `